File: HTTP_Server_CLI/scenarios/php/1.1.3-static_100.php
<?php
// @label: 100 static routes
// @group: Static routes
// @competitors: all
// Round-robins through all 100 registered static routes.

$paths = ['/', '/about', '/contact', '/blog', '/pricing', '/docs', '/faq', '/terms', '/privacy', '/status'];
for ($i = 11; $i <= 100; $i++) {
   $paths[] = "/static/{$i}";
}

return [
   'method' => 'GET',
   'paths'  => $paths,
];

File: HTTP_Server_CLI/scenarios/1.1.3-static_100.lua
-- @label: 100 static routes
-- @group: Static routes
-- @competitors: all
-- Round-robins through all 100 registered static routes.

local paths = {"/", "/about", "/contact", "/blog", "/pricing", "/docs", "/faq", "/terms", "/privacy", "/status"}
for i = 11, 100 do
   paths[#paths + 1] = "/static/" .. i
end

local counter = 0

request = function()
   counter = counter + 1
   return wrk.format("GET", paths[(counter % #paths) + 1])
end
